src/views/ApplicationPage/index.js

```diff
@@ -72,40 +83,84 @@ export default class application extends Component {
     console.log("Submitting!");
     event.preventDefault();
     if (
-      this.state.Gender == undefined ||
+      this.state.gender == undefined ||
       this.state.ethnicity == undefined ||
-      this.state.schoolOption == undefined ||
-      this.state.Degree == undefined ||
-      this.state.Graduation_year == undefined ||
-      this.state.Program == undefined
+      this.state.school == undefined ||
+      this.state.degree == undefined ||
+      this.state.graduation_year == undefined ||
+      this.state.program == undefined ||
+      this.state.why_goldenHack == undefined
     ) {
       document.getElementById("display_error").innerHTML =
         "Not all required fields have been filled out.";
+      return;
     }
-    if (!this.state.Github_URL.startsWith("https://www.github.com/")) {
+    if (
+      this.state.github_url != undefined &&
+      !this.state.github_url.startsWith("https://www.github.com/")
+    ) {
       document.getElementById("display_error").innerHTML =
         "Invalid URL entered";
     }
-    if (!this.state.LinkedIn_URL.startsWith("https://www.linkedin.com/in/")) {
+    if (
+      this.state.linkedin_url != undefined &&
+      !this.state.linkedin_url.startsWith("https://www.linkedin.com/in/")
+    ) {
       document.getElementById("display_error").innerHTML =
         "Invalid URL entered";
     }
-    if (!this.state.Dribbble_URL.startsWith("https://www.dribbble.com/")) {
+    if (
+      this.state.dribbble_url != undefined &&
+      !this.state.dribbble_url.startsWith("https://www.dribbble.com/")
+    ) {
       document.getElementById("display_error").innerHTML =
         "Invalid URL entered";
     }
-    if (!this.state.Link_to_resume.startsWith("https://")) {
+    if (
+      this.state.link_to_resume != undefined &&
+      !this.state.link_to_resume.startsWith("https://")
+    ) {
       document.getElementById("display_error").innerHTML =
         "Invalid URL entered";
     }
-    if (this.state.Why_GoldenHack.length > 1000) {
+    if (this.state.why_goldenHack.length > 1000) {
       document.getElementById("display_error").innerHTML =
         "Over 1000 character limit for 'Why Golden Hack'";
     }
 
     this.setState({ submitted: true });
 
     console.log(this.state);
+    let docClient = new AWS.DynamoDB.DocumentClient();
+    let input = {
+      email: "dh", // How do i get the email from JWT
```

Comment:
Take a look at how it was done here -> https://github.com/thegoldenhack/website-2020/blob/94f170cab16a7f777f12763bc69693832be89efa/src/views/DashboardPage/index.js#L56